logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/register/RegisterAccountUseCase.kt

```diff
@@ -0,0 +1,65 @@
+package com.wire.kalium.logic.feature.register
+
+import com.wire.kalium.logic.NetworkFailure
+import com.wire.kalium.logic.configuration.ServerConfig
+import com.wire.kalium.logic.data.register.RegisterAccountRepository
+import com.wire.kalium.logic.feature.auth.AuthSession
+import com.wire.kalium.logic.feature.auth.AuthenticationResult
+import com.wire.kalium.logic.feature.auth.LoginUseCase
+import com.wire.kalium.logic.functional.suspending
+
+sealed class RegistrationParam(
```

Comment:
All the registration related classes have Register* naming pattern, do you think it's better to apply that name pattern here? `RegisterParam` or rename other to this pattern `RegistrationResult`.